client/blocks/site/index.jsx

```diff
@@ -11,10 +11,13 @@ import Gridicon from 'gridicons';
  */
 import SiteIcon from 'blocks/site-icon';
 import SiteIndicator from 'my-sites/site-indicator';
+import PureRenderMixin from 'react-pure-render/mixin';
```

Comment:
this was part of old change, since then I've found that upgrading `react-redux` to version 5 solve the issue as it cut the rendering inside the `connectHOC` so there is actually no need to make the visual component pure.